TFT/src/User/Configuration.h

```diff
@@ -484,25 +486,74 @@
  * Pause/Nozzle Park Settings
  * These settings are used when a print is paused or in any feature which requires moving/parking the nozzle
  * before performing a task like in (Un)Load or Extruder Tuning menus.
+ *
+ * Pause Retract Length
+ *   Format: [pause_retract: R<retract length> P<resume purge length>]
+ *   Unit: [length in mm]
+ *   Value range: [min: 0.0, max: 20.0]
+ *
+ * Pause XY Position
+ * NOTES:
+ *   - It MUST BE a value >= 0 for a Cartesian printer.
+ *   - It MUST BE a value <= 0 for a Delta printer.
+ *
+ *   Format: [pause_pos: X<position> Y<position>]
+ *   Unit: [position in mm]
+ *   Value range: [min: -2000.0, max: 2000.0]
+ *
+ * Pause Z Raise
+ * Raise Z axis by this value relative to the current layer height.
+ *   Unit: [distance in mm]
+ *   Value range: [min: 0.0, max: 2000.0]
+ *
+ * Pause Feed Rate
+ * Feedrate to use when moving an axis when printing is paused.
+ *   Format: [pause_feedrate: XY<feedrate> Z<feedrate> E<feedrate>]
+ *   Unit: [feedrate in mm/min]
+ *   Value range: [min: 10, max: 12000]
  */
-#define NOZZLE_PAUSE_RETRACT_LENGTH               15.0f  // (mm) (Default: 15.0f)
-#define NOZZLE_RESUME_PURGE_LENGTH                16.0f  // (mm) (Default: 16.0f)
-#define NOZZLE_PAUSE_X_POSITION     (X_MIN_POS + 10.0f)  // (mm) Must be an integer (Default: 10.0f)
-#define NOZZLE_PAUSE_Y_POSITION     (Y_MIN_POS + 10.0f)  // (mm) Must be an integer (Default: 10.0f)
-#define NOZZLE_PAUSE_Z_RAISE                      10.0f  // (mm) (Default: 10.0f)
-#define NOZZLE_PAUSE_XY_FEEDRATE                   6000  // (mm/min) X and Y axes feedrate (Default: 6000)
-#define NOZZLE_PAUSE_Z_FEEDRATE                    6000  // (mm/min) Z axis feedrate (Default: 6000)
-#define NOZZLE_PAUSE_E_FEEDRATE                     600  // (mm/min) retract & purge feedrate (Default: 600)
```

Comment:
you should not remove the unit names.

Reply:
they are reported in the comment above (as in config.ini)

Reply:
The description above is wrong. 
you are explaining how to set the values in the **configuration.h** file, but the details are tailored for values **config.ini** file.
Like: `Format: [pause_feedrate: XY<feedrate> Z<feedrate> E<feedrate>]` does not apply to **configuration.h**

Reply:
yes I know that. There are other params like that but it seems quite clear how they are related each other and with config.ini. Just to not waste too much time on that

Reply:
The existing descriptions were similar to that in marlin firmware.
![image](https://user-images.githubusercontent.com/22422544/141340884-6f43bb53-e3cd-4602-84fe-4aaeea1a234f.png)

Reply:
ah ok. it is not a problem to restore the previous description for that block of param (pause and level)

Reply:
restored the old inline description